packages/moonstone/DatePicker/DatePickerBase.js

```diff
@@ -210,13 +246,16 @@ const DatePickerBase = kind({
 	},
 
 	render: ({
+		['aria-label']: ariaLabel,
```

Comment:
The brackets aren't required here; quoting the prop name is sufficient. There are a few other instances of this as well.

```
render: ({
  'aria-label': ariaLabel,
```

Reply:
I will apply this.